lock-api/src/main/java/com/palantir/lock/LockRequest.java

```diff
@@ -48,8 +50,17 @@
 @Immutable public final class LockRequest implements Serializable {
     private static final long serialVersionUID = 0xf6c12b970b44af68L;
 
+    private static final AtomicReference<TimeDuration> DEFAULT_LOCK_TIMEOUT = new AtomicReference<>(
+            SimpleTimeDuration.of(120, TimeUnit.SECONDS));
+
     /** The default amount of time that it takes a lock (lease) to expire. */
-    public static final TimeDuration DEFAULT_LOCK_TIMEOUT = SimpleTimeDuration.of(120, TimeUnit.SECONDS);
+    public static TimeDuration getDefaultLockTimeout() {
+        return DEFAULT_LOCK_TIMEOUT.get();
+    }
+
+    public static void setDefaultLockTimeout(TimeDuration timeout) {
+        DEFAULT_LOCK_TIMEOUT.set(timeout);
```

Comment:
should probably sanity check timeout is non-negative and define what timeout of 0 means